src/hci/gnomex/httpclient/SaveChromatogramsFromFiles.java

```diff
@@ -39,7 +39,7 @@
 private StringBuffer errors;
 private int numerrors;
 private boolean noemail = false;
-private String toAddress = "Tim.Maness@hci.utah.edu"; // "LabTech_Group@genetics.utah.edu";
+private String toAddress = "Tim.Maness@hci.utah.edu;LabTech_Group@genetics.utah.edu";
```

Comment:
will you add gnomex.support@hci.utah.edu to this so in case you are out someone else will be notified.  Thanks.